src/Cake.Common/Build/AppVeyor/IAppVeyorProvider.cs

```diff
@@ -1,33 +0,0 @@
-﻿using Cake.Common.Build.AppVeyor.Data;
-using Cake.Core.IO;
-
-namespace Cake.Common.Build.AppVeyor
-{
-    /// <summary>
-    /// Represents a service that communicates with AppVeyor.
-    /// </summary>
-    public interface IAppVeyorProvider
```

Comment:
Would actually want to keep the interface for the AppVeyor provider since it allows mocking. Should be injected into to BuildSystem. Would also be nice if there was an ITeamCityProvider.